File: src/ois_seconddose/seconddose.pas
{ NOTE: it is recommended to use this even if you don't understand the following code. }

{ constraints }
const
    MAXN = 100000;

{ input data }
var
    N1, N2, i  : longint;
    ID1, ID2   : array[0..MAXN-1] of longint;

begin
{
    uncomment the following lines if you want to read/write from files
    assign(input,  'input.txt');  reset(input);
    assign(output, 'output.txt'); rewrite(output);
}

    readln(N1, N2);
    for i:=0 to N1-1 do
        read(ID1[i]);
    readln();
    for i:=0 to N2-1 do
        read(ID2[i]);
    readln();

    { insert your code here }

    writeln(42); { print result }
end.

File: src/ois_seconddose/seconddose.c
#include <stdio.h>
#include <assert.h>

// constraints
#define MAXN 100000

// input data
int N1, N2, i;
int ID1[MAXN], ID2[MAXN];

int main() {
//  uncomment the following lines if you want to read/write from files
//  freopen("input.txt", "r", stdin);
//  freopen("output.txt", "w", stdout);

    assert(2 == scanf("%d %d", &N1, &N2));
    for(i=0; i<N1; i++)
        assert(1 == scanf("%d", &ID1[i]));
    for(i=0; i<N2; i++)
        assert(1 == scanf("%d", &ID2[i]));

    // insert your code here
    
    printf("%d\n", 42); // print the result
    return 0;
}
